projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -890,4 +899,21 @@ private function is_user_connected() {
 	private function get_site_id() {
 		return Connection_Manager::get_site_id();
 	}
+
+	/**
+	 * Check if the posts are synced.
+	 *
+	 * @return bool True if is sync, false otherwise.
+	 */
+	private function is_posts_synced(): bool {
+
```

Comment:
My understanding based on `is_user_connected` is that this endpoint is also available for WPCOM Simple sites. If that's the case, let's add an early return, since Sync is not present (and makes no sense) for Simple sites.
```suggestion
		// On WordPress.com Simple, Sync is not present.
		if ( ( new Host() )->is_wpcom_simple() ) {
			return false;
		}
```